src/applications/financial-status-report/components/monetary/CashOnHand.jsx

```diff
@@ -33,6 +34,16 @@ const CashOnHand = ({
         cashOnHand: cash,
       },
     });
+
+    // update gmtData for isStreamlinedShortForm
+    setFormData({
+      ...data,
+      gmtData: {
+        ...gmtData,
+        cashBelowGmt: safeNumber(cash) < gmtData?.assetThreshold,
+      },
+    });
+
```

Comment:
maybe we can combine this into a single setFormData?

```
setFormData({
  ...data,
  assets: {
    ...data.assets,
    cashOnHand: cash,
  },
  gmtData: {
    ...gmtData,
    cashBelowGmt: safeNumber(cash) < gmtData?.assetThreshold,
  },
});
```

Reply:
Updated accordingly! Thanks for the input!